pom.xml

```diff
@@ -66,7 +66,7 @@
     <spark2.version>2.1.3</spark2.version>
     <hydrator.version>2.10.0</hydrator.version>
     <commons.version>3.8.1</commons.version>
-    <salesforce.api.version>53.0.0</salesforce.api.version>
+    <salesforce.api.version>61.0.0</salesforce.api.version>
```

Comment:
raise a separate PR just for this version upgrade.

Reply:
https://github.com/data-integrations/salesforce/pull/255